INSTALL.md

```diff
@@ -38,10 +38,22 @@ Dangerzone is available for:
   ```
 </details>
 
-Add our repository following [these instructions](https://packagecloud.io/firstlookmedia/code/install#manual-deb), or by running this script:
-
-```
-curl -s https://packagecloud.io/install/repositories/firstlookmedia/code/script.deb.sh | sudo bash
+Add our repository following these instructions:
+
+```sh
+sudo apt-get update
+sudo apt-get install -y curl
+sudo mkdir -p /etc/apt/keyrings/
+curl -S https://packages.freedom.press/apt-tools-prod/fpf-archive-keyring.gpg \
```

Comment:
Won't we need to `gpg --dearmor` this?

Reply:
If you look at `apt-tools-prod`, the key should be in DER format, so no need to dearmor it.